blueflood-http/src/integration-test/java/com/rackspacecloud/blueflood/inputs/handlers/HttpAnnotationsEndToEndTest.java

```diff
@@ -73,7 +73,7 @@ public static void setUp() throws Exception {
         esSetup.execute(EsSetup.deleteAll());
         esSetup.execute(EsSetup.createIndex(EventElasticSearchIO.EVENT_INDEX)
                 .withSettings(EsSetup.fromClassPath("index_settings.json"))
-                .withMapping("metrics", EsSetup.fromClassPath("events_mapping.json")));
+                .withMapping("graphite_event", EsSetup.fromClassPath("events_mapping.json")));
         eventsSearchIO = new EventElasticSearchIO(esSetup.client());
```

Comment:
Note here that the document type to retrieve from events_mapping.json is actually called "graphite_event", not "metrics" 
(see https://github.com/rackerlabs/blueflood/blob/master/blueflood-elasticsearch/src/main/resources/events_mapping.json).

This was causing weird errors to occur later on, not at the time of esSetup execute for creating the index, because of the flexible nature of elastic search.